dev/devicelab/bin/tasks/build_ios_framework_module_test.dart

```diff
@@ -66,7 +66,7 @@ Future<void> _testBuildIosFramework(Directory projectDir, { bool isModule = fals
   String content = pubspec.readAsStringSync();
   content = content.replaceFirst(
     '\ndependencies:\n',
-    '\ndependencies:\n  device_info: 0.4.1\n  package_info: 0.4.0+9\n  connectivity: 3.0.3\n',
+    '\ndependencies:\n  device_info: 2.0.3\n  package_info: 0.4.0+9\n  connectivity: 3.0.3\n',
```

Comment:
It continues... `connectivity` should be at least 3.0.4 https://github.com/flutter/plugins/pull/3909

The log is complaining about `package_info` but it's already past 2.0.1 https://github.com/flutter/plugins/pull/3918, guessing it's because `rootProject.allprojects` is polluting the project.

https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8825198889713473617/+/u/run_module_test/test_stdout
```
[module_test] [STDOUT] stderr: FAILURE: Build failed with an exception.
[module_test] [STDOUT] stderr: 
[module_test] [STDOUT] stderr: * What went wrong:
[module_test] [STDOUT] stderr: A problem occurred configuring project ':package_info'.
```

Reply:
```suggestion
    '\ndependencies:\n  device_info: 2.0.3\n  package_info: 0.4.0+9\n  connectivity: 3.0.6\n',
```